packages/cli/src/commands/new.ts

```diff
@@ -66,7 +67,29 @@ export class New extends Command {
         {name: "Formik"},
       ]
 
-      const promptResult: any = await this.enquirer.prompt({
+      const databaseChoices: Array<{name: AppGeneratorOptions["database"]; message?: string}> = [
+        {name: "SQLite", message: "SQLite (default)"},
+        {name: "PostgreSQL"},
+      ]
+
+      const databasePromptResult: any = await this.enquirer.prompt({
+        type: "select",
+        name: "database",
+        message: "Pick a database engine (you can switch to something else later if you want)",
+        choices: databaseChoices,
+      })
+
+      const databaseUrlPromptResult: any = await this.enquirer.prompt({
+        type: "input",
+        name: "databaseUrl",
+        message: `Specify url for ${databasePromptResult.database} (you can change to something else later if you want)`,
+        initial:
+          databasePromptResult.database === "SQLite"
+            ? "file:./db.sqlite"
+            : `postgresql://${await username()}@localhost:5432/${appName}`,
+      })
```

Comment:
Let's skip this prompt, and instead just set the URL automatically like we have been. The reason is the vast majority of folks won't want to change this.

Reply:
The reason I added this is when you select `PostgreSQL` the default URL is almost always wrong, for example on my dev setup I always have `postgres@localhost` this is the default user used by PostgreSQL. I think allowing to change this is quite a nice touch and if you want to leave the default one you can just press <Enter> as it suggests you the right one.

Reply:
I just realized yesterday that the username thing is unnecessary and yes the default username should be `postgres`, so let's make that change. And then do you think it's fine to leave out this prompt?

Reply:
Yes I think it would be enough, for now, I will update the code 🙂